### DISCUSSION TITLE
Production-ready ?

### DISCUSSION THREAD

**Discussion by HeyGuyListenToMe** [NativeScript/NativeScript#10543] [General]
Hello,

Is NativeScript production-ready ?
Is it dead ?
Has it correct performance on Android and iOS phones ?

I'm afraid to use it...
For example, I have just trying to set up NS for Linux and when I run the curl command to install Node.js 15 I noticed that the version is no longer actively supported... So the documentation isn't up-to-date

Thank you

**Comment by rigor789** [comment]
We use it daily, and across a variety of production apps.

Here's a recent one (we are still working on)
https://nstudio.io/success-stories/blackout-lighting-console

The above should give you an idea on performance, it's a fairly complex app.

**Reply by HeyGuyListenToMe** [reply]
Why did you choose NativeScript ?
I'm afraid to use NativeScript, it's not famous like RN, Flutter, ...etc

**Reply by rigor789** [reply]
I really enjoy writing apps in Vue, and none of the other alternatives allow me to do that... (Only via webview). I also firmly believe in the fundamentals NativeScript is built upon. Plus the community is great, and welcoming.

**Comment by vallemar** [comment]
Hi! Yes, I use it for a production application without problems (I use it with vue3).

Node.js 15 is quite old (I don't know if it should work or not), try at least node18 or 20 (current). 
Yes, someone should check out the linux page, you can also submit a PR if you are interested in contributing!

**Comment by NathanWalker** [comment]
It always delivers great production results for us at [nstudio](https://nstudio.io). It's fundamentally flexible based on how it was designed so it never limits any tech option needed to achieve a production goal.
Some helpful things to check can be found here:
https://x.com/NativeScript/status/1784704643463028970

Chat with a few folks in the community discord to dive deeper into questions: [nativescript.org/discord](https://t.co/C4oo7v6xaG)
